handlers/aws/utils.py

```diff
@@ -302,7 +301,7 @@ def get_trigger_type_and_config_source(event: dict[str, Any]) -> tuple[str, str]
                 and "output_args" in event_body
                 and "event_payload" in event_body
             ):
-                return "replay-sqs", CONFIG_FROM_PAYLOAD
+                return "replay-sqs", CONFIG_FROM_S3FILE
```

Comment:
it does not make really sense to load the config from the payload in the case of the replay queue.

the idea of the config being read from the s3 file or the payload in the continuing queue is given by the fact that a continuation happens when a trigger payload was not able to be fully forwarded: we somehow postpone finishing the execution of the trigger in the future, where the s3 config file might change but we want to ensure that we will still execute with the original one

but in the case of the replay queue we are reacting on a forwarding failure: this might be indeed because of "wrong" information (like credentials, or URLs) contained in the s3 config file, that lead the forwarding to fail in the first place.
the replay queue is a failure recovery scenario: having the replay queue as an enabled trigger of the forwarder lambda is an explicit intention of the users, if they are not given the option to upload a new "fixed" s3 config file we prevent them to apply the recovery conditions